tools/build_hooks/REALTEK_RTL8195AM.py

```diff
@@ -0,0 +1,144 @@
+"""
```

Comment:
The official place for post build hooks is now `tools/targets/`. Could you move this file to this location?